css/amp-story-player-shadow.css

```diff
@@ -32,7 +32,7 @@
 
 .i-amphtml-story-player-main-container iframe:nth-of-type(1),
 .i-amphtml-story-player-main-container .story-player-iframe[i-amphtml-iframe-position="0"] {
-  transform: translate3d(0, 0, 1px);
+  transform: translate3d(0, 0, 0);
 }
```

Comment:
Do we know that the active iframe will still stay on top of others if we make this change (as that looks like the original intention of this code)?  We should test both initial load and cross-browser cases